tt_metal/impl/dispatch/kernels/cq_prefetch.cpp

```diff
@@ -1397,8 +1406,10 @@ void kernel_main_hd() {
     uint32_t fence = cmddat_q_base;
     bool done = false;
     uint32_t heartbeat = 0;
+    uint32_t l1_cache[l1_cache_elements_rounded];
+    PrefetchExecBufState exec_buf_state;
     while (!done) {
-        DeviceZoneScopedND("KERNEL-MAIN-HD", block_noc_writes_to_clear, rd_block_idx );
+        DeviceZoneScopedN("KERNEL-MAIN-HD");
```

Comment:
As per our slack conv. , let's keep this unchanged. This doesn't do quick_push which flushes L1 to DRAM without going to FW

Reply:
I've removed this change for now. Lets revisit this once we have quick_push APIs that dont need the `block_noc_writes_to_clear` and `rd_block_idx` params

Reply:
sounds good, yes https://github.com/orgs/tenstorrent/projects/26/views/1?pane=issue&itemId=70803557 will revisit this.